app/controllers/admin/suppliers/lead_provider_users_controller.rb

```diff
@@ -16,6 +16,12 @@ def update
         end
       end
 
+      def destroy
+        authorize @lead_provider_user
+        @lead_provider_user.lead_provider_profile.discard!
+        redirect_to admin_supplier_users_path(user_deleted: true)
+      end
+
```

Comment:
We probably want `delete` method authorising things too - like in the admin controller

Reply:
Done